easybuild/easyconfigs/m/Mathematica/Mathematica-12.0.0.eb

```diff
@@ -12,6 +12,8 @@ checksums = ['b9fb71e1afcc1d72c200196ffa434512d208fa2920e207878433f504e58ae9d7']
 
 license_server = 'license.example.com'
 
-sanity_check_commands = [('mathematica', '--version')]
+postinstallcmds = ['ln -s %(installdir)s/Executables/wolframscript %(installdir)s/bin/']
+
+sanity_check_commands = ['mathematica --version']
```

Comment:
@smoors This belongs in the easyblock too? And so does the symlinking you're now doing via `postinstallcmds`?

Reply:
@boegel I was not sure if the symlink should go here or in the easyblock. I decided to treat it as a bug fix (like a patch), because in version 11 the wolframscript symlink is present. I expect that it will be fixed again in the next version, so to prevent having to update the easyblock again I did it here.

does this make sense to you? no problem for me if you still want it in the easyblock.

Reply:
nevermind, I think I have a better way to fix this

Reply:
You can check if the path exists, and if it doesn't, symlink it?

Reply:
I decided to just add Executables to the PATH (see easybuilders/easybuild-easyblocks#2052), since everything in bin is symlinking to Executables anyway.